transip/service/vps.py

```diff
@@ -0,0 +1,252 @@
+""" The connector to VPS related API calls """
+
+from transip.client import Client, MODE_RO, MODE_RW
+
+class VpsService(Client):
+    """ Representation of the VpsService API calls for TransIP """
+
+    def __init__(self, *args, **kwargs):
+        super(VpsService, self).__init__('VpsService', *args, **kwargs)
+
+    def get_available_products(self):
+        """ Get available VPS products """
+        cookie = self.build_cookie(mode=MODE_RO, method='getAvailableProducts')
+        self.update_cookie(cookie)
+        return self.soap_client.service.getAvailableProducts()
+
+    def get_available_addons(self):
+        """ Get available VPS addons """
+        cookie = self.build_cookie(mode=MODE_RO, method='getAvailableAddons')
+        self.update_cookie(cookie)
+        return self.soap_client.service.getAvailableAddons()
+
+    def get_active_addons_for_vps(self, vps_name):
+        """ Get all the Active Addons for Vps """
+        cookie = self.build_cookie(mode=MODE_RO, method='getActiveAddonsForVps', parameters=[vps_name])
+        self.update_cookie(cookie)
+        return self.soap_client.service.getActiveAddonsForVps(vps_name)
+
+    def get_available_upgrades(self, vps_name):
+        """ Get available VPS upgrades for a specific Vps """
+        cookie = self.build_cookie(mode=MODE_RO, method='getAvailableUpgrades', parameters=[vps_name])
+        self.update_cookie(cookie)
+        return self.soap_client.service.getAvailableUpgrades(vps_name)
+
+    def get_available_addons_for_vps(self, vps_name):
+        """ Get available Addons for Vps """
+        cookie = self.build_cookie(mode=MODE_RO, method='getAvailableAddonsForVps', parameters=[vps_name])
+        self.update_cookie(cookie)
+        return self.soap_client.service.getAvailableAddonsForVps(vps_name)
+
+    def get_cancellable_addons_for_vps(self, vps_name):
+        """ Get cancellable addons for specific Vps """
+        cookie = self.build_cookie(mode=MODE_RO, method='getCancellableAddonsForVps', parameters=[vps_name])
+        self.update_cookie(cookie)
+        return self.soap_client.service.getCancellableAddonsForVps(vps_name)
+
+    # def order_vps(self, product_name, addons, operating_system_name, hostname):
```

Comment:
Can you add a comment that describes why this is commented out? It helpful to have this documented to so that people can know why this isn't currently supported. Thanks.